### PR TITLE
Create Queue and Join Queue section redesign

### PR DESCRIPTION
**Pull request by KasiKral**
#### What does this implement/fix? Explained changes.
I redesigned Create Queue and Join Queue "How To" Section according to "Web: Home – 2" design.
#### Any other comments?
- I exported icons from a design.
- I removed slider component and made changes to the layout of  Create Queue and Join Queue section.
Changes should be fully responsible. If you think there should be other changes let me know.

### CODE REVIEW

**Comment by daltonfury42** [comment]
Hi @KasiKral Thanks for the PR. Can you pull our latest code and resolve the conflicts?

File: simplq/src/components/pages/Home/StaticInfos.jsx [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import React from 'react';
 import styles from '../../../styles/homePage.module.scss';
-import { CreatorSlider, JoinerSlider } from '../../common/slider';
```
Comment (daltonfury42) [inline simplq/src/components/pages/Home/StaticInfos.jsx:3]: Let's delete the slider folder as well. And remove `react-awesome-slider` from package.json

Comment (KasiKral) [reply simplq/src/components/pages/Home/StaticInfos.jsx:3]: Can u help me when I want to commit I get this.
image
What should I do?

Comment (maaverik) [reply simplq/src/components/pages/Home/StaticInfos.jsx:3]: Hi, add these two lines to the top of Burger.jsx to let eslint know you want to suppress these warnings for now.

```
/* eslint-disable jsx-a11y/click-events-have-key-events */
/* eslint-disable jsx-a11y/no-static-element-interactions */

```

We'll get to these later.

Comment (maaverik) [reply simplq/src/components/pages/Home/StaticInfos.jsx:3]: This should let you commit fine

Comment (KasiKral) [reply simplq/src/components/pages/Home/StaticInfos.jsx:3]: Thanks for the help!

**Review by maaverik — APPROVED** [review]

File: simplq/src/styles/homePage.module.scss [hunk 2]
```diff
@@ -101,11 +101,64 @@
...
+  .queue-container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    width: 100%;
+    padding: 0;
+    .queue {
+      flex: 0 1 400px;
+      text-align: left;
+      margin: 2rem;
```
Comment (maaverik) [inline simplq/src/styles/homePage.module.scss:123]: Can you stick replace px with rem everywhere in the SCSS files? We're sticking to rem everywhere else and keeping that consistent would be best.

Comment (KasiKral) [reply simplq/src/styles/homePage.module.scss:123]: Okay, I'll do that. I see now that something is wrong with navigation should I try to fix it?

Comment (maaverik) [reply simplq/src/styles/homePage.module.scss:123]: If you can, then please go ahead. But, what's the problem?

Comment (KasiKral) [reply simplq/src/styles/homePage.module.scss:123]: image
It looks like this on most phones.

Comment (daltonfury42) [reply simplq/src/styles/homePage.module.scss:123]: I think I've seen this before when you open a page, and then switch to mobile view. Try refreshing and see if it goes away.

If that's the case, a real user would not switch between desktop <-> mobile, and this is fine.

Comment (daltonfury42) [reply simplq/src/styles/homePage.module.scss:123]: Let's switch to `rem` and this is good to merge.

Comment (KasiKral) [reply simplq/src/styles/homePage.module.scss:123]: I changed values but after refreshing the navbar hasn't changed.

**Review by daltonfury42 — APPROVED** [review]

File: simplq/src/styles/homePage.module.scss (hunk 2 above)
Comment (daltonfury42) [reply simplq/src/styles/homePage.module.scss:123]: Let me merge your changes and see if I can reproduce this issue. 🤔

**Pull request merged**